python-avocado.spec

```diff
@@ -1,8 +1,24 @@
 %global srcname avocado
+
+# Conditional for release vs. snapshot builds. Set to 1 for release build.
+%if ! 0%{?rel_build:1}
+%global rel_build 1
+%endif
+
+# Settings used for build from snapshots.
+%if 0%{?rel_build}
+%global gittar		%{srcname}-%{version}.tar.gz
+%else
 %if ! 0%{?commit:1}
- %define commit 1d717446d28ff852fe5de895f14c7985c63ada3e
+%global commit		817c25c2bbedb6d43e5fa2ad0a78698f1416ab49
+%endif
+%if ! 0%{?commit_date:1}
+%global commit_date	20170407
+%endif
+%global shortcommit	%(c=%{commit};echo ${c:0:7})
+%global gitrel		.%{commit_date}git%{shortcommit}
+%global gittar		%{srcname}-%{shortcommit}.tar.gz
```

Comment:
Btw is it not? https://fedoraproject.org/wiki/PeterGordon/SpecFormattingGuidelines (the example is not indented, but the description says it should be. And googling I found several patches to use indentation to make visually clear the nested conditional blocks... Anyway let's sync the implementations and then we can try to improve it both, up and downstream....